FEXCore/Source/Interface/IR/Passes/x87StackOptimizationPass.cpp

```diff
@@ -737,12 +752,22 @@ void X87StackOptimization::Run(IREmitter* Emit) {
       }
 
       case OP_F80SINSTACK: {
-        HandleUnop(OP_F64SIN, false, OP_F80SIN);
+        Ref St0 = LoadStackValue();
```

Comment:
Could the below 5 cases be handled simpler in the interpreter fallback itself? Or does this allow constprop to remove checks

Reply:
Ah yes, but then the IR will lack specificity which aids debugging quite a bit. Unless there's a real reason to move these away from the pass, I would prefer to keep them here.

Reply:
I think I addressed your concerns and things are looking much better. Of course, a lot of the inflation of instcountci is now hidden because the checks run in the fallbacks, but it looks much simpler so that seems like a plus. Thanks for the time helping out on this with your reviews.